File: BackEndCrudAngular.Server/Models/Empleado.cs
using Microsoft.Data.SqlClient;

namespace BackEndCrudAngular.Server.Models
{
    public class Empleado
    {
        public int intId { get; set; }
        public int intDepartamentoId { get; set; }
        public string strNombreCompleto { get; set; }
        public string? strEstado { get; set; }
        public string strCorreo { get; set; }
        public decimal dcSalario { get; set; }

        public DateTime? dtFechaIngreso { get; set; }
        public DateTime? dtFechaModificacion { get; set; }

       
    }
}

File: FrontEndCrudAngular.client/src/app/Models/Empleado.ts
export interface Empleado 
{
    intId: number,
	strNombreCompleto:string,
	intDepartamentoId:number,
	strEstado:string,
	strCorreo:string,
	dcSalario:number,
    dtFechaIngreso:Date,
    dtFechaModificacion:Date | null
}